wafer/static/css/wafer.css

```diff
@@ -70,6 +70,19 @@ ul.profile-links li {
     background-color: white;
 }
 
+#wafer-edit-schedule-bucket .nav-tabs a.active {
+  font-weight: bold;
+}
```

Comment:
These tabs still look pretty awful. It'd be really nice to put them in a visual tab / block of some sort.

![image](https://user-images.githubusercontent.com/442117/89108322-4a450000-d3ec-11ea-9906-00bc9d64b978.png)

However, this is certainly an improvement, so won't block this PR on that.

Reply:
fixed with some spacing.

![bucket-nav](https://user-images.githubusercontent.com/30027/89110117-ef2c0100-d41d-11ea-9cd3-bf521328d96b.png)

I tried to make it look like actual tabs, but I couldn't figure out how yet. I think a good chunk of that html needs to be rewritten.

Reply:
Probably :( And this sections below factored out, because there's a lot of duplication.